src/components/Popups/index.tsx

```diff
@@ -65,16 +67,20 @@ export default function Popups() {
   const { chainId } = useActiveWeb3React()
   const isNotOnMainnet = Boolean(chainId && chainId !== SupportedChainId.MAINNET)
 
+  const [showDonation] = useShowDonationLink()
+
   return (
     <>
       <FixedPopupColumn gap="20px" extraPadding={urlWarningActive} xlPadding={isNotOnMainnet}>
         <ClaimPopup />
         {activePopups.map((item) => (
```

Comment:
not familiar with `activePopups` but could yo not leverage that here instead of doing a custom donation popup?

Reply:
i think for custom styling and link we'd need a custom component here - unless we wanted to change design to work with txn popups (which have a text string and a link)